tests/CMakeLists.txt

```diff
@@ -28,12 +28,13 @@ set( CMAKE_CXX_STANDARD 14 )
 
 
 file(GLOB UNIT_TESTS "chain_tests/*.cpp")
+file(GLOB UNIT_TESTS "tests/abi_tests.cpp")
```

Comment:
this should be merged with the line above.  `file(GLOB ...` can accept multiple patterns.  See: https://cmake.org/cmake/help/latest/command/file.html